spec/features/dashboards_spec.rb

```diff
@@ -29,6 +29,24 @@
     end
   end
 
+  describe "static dashboards" do
+    scenario "creating a static dashboard" do
+      dashboard = FactoryGirl.create(:dashboard)
+      visit dashboard_slug_path dashboard.slug
+      expect {
+        find('.glyphicon-link').click
```

Comment:
Don't have a much better idea, but I'd at least search only in `#global_controls` for this glyphicon.